pkg/controller/configmapcainjector/controller/configmap_injecting_controller.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/openshift/service-ca-operator/pkg/boilerplate/controller"
 	"github.com/openshift/service-ca-operator/pkg/controller/api"
+	"github.com/openshift/library-go/pkg/operator/certmonitor"
```

Comment:
The change moved to `github.com/openshift/library-go/pkg/operator/certrotation/metrics`